engine/core/src/main/java/org/datacleaner/metamodel/datahub/DataHubDataSet.java

```diff
@@ -158,6 +160,10 @@ private String createEncodedUri(DataHubConnection connection, Table table) {
     }
 
     private String getQueryString(Query query, Table table) {
+        final OrderByClause orderByClause = query.getOrderByClause();
+        if(orderByClause.isEmpty()) {
```

Comment:
beware that not all queries _can_ have an order by clause. For instance a "SELECT COUNT(*) from table" cannot have an appended order by